package.json

```diff
@@ -5,19 +5,20 @@
   "license": "BSD-3-Clause",
   "repository": "https://github.com/GPII/gpii-binder",
   "scripts": {
+    "pretest": "node_modules/rimraf/bin.js coverage instrumented",
```

Comment:
I updated it to use `node node_modules/rimraf/bin.js`, which should work anywhere.